tests/tools/dumping/test_processes.py

```diff
@@ -266,14 +275,31 @@ def test_dump_calculation_flat(tmp_path, generate_calculation_node_io):
 
 
 # Here, in principle, test only non-default arguments, as defaults tested above
-# @pytest.mark.parametrize('overwrite', (True, False))
-def test_dump_calculation_overwrite(tmp_path, generate_calculation_node_io):
+def test_dump_calculation_overwr_incr(tmp_path, generate_calculation_node_io):
     dump_parent_path = tmp_path / 'cj-dump-test-overwrite'
-    process_dumper = ProcessDumper(overwrite=False)
+    process_dumper = ProcessDumper(overwrite=False, incremental=False)
     calculation_node = generate_calculation_node_io()
-    process_dumper._dump_calculation(calculation_node=calculation_node, output_path=dump_parent_path)
+    calculation_node.seal()
+    # Create safeguard file to mock existing dump directory
+    dump_parent_path.mkdir()
+    (dump_parent_path / '.aiida_node_metadata.yaml').touch()
     with pytest.raises(FileExistsError):
         process_dumper._dump_calculation(calculation_node=calculation_node, output_path=dump_parent_path)
+    # This just doesn't raise
+    process_dumper = ProcessDumper(overwrite=True, incremental=False)
+    process_dumper._dump_calculation(calculation_node=calculation_node, output_path=dump_parent_path)
+    assert (dump_parent_path / inputs_relpath / filename).is_file()
+
+    shutil.rmtree(dump_parent_path)
+
+    # Incremental also does work
+    dump_parent_path.mkdir()
+    (dump_parent_path / '.aiida_node_metadata.yaml').touch()
+    process_dumper = ProcessDumper(overwrite=False, incremental=True)
+    process_dumper._dump_calculation(calculation_node=calculation_node, output_path=dump_parent_path)
+    assert (dump_parent_path / inputs_relpath / filename).is_file()
+
+    # Having both, overwrite and incremental True -> Directory gets cleaned and then everything dumped
```

Comment:
Is this some leftover comment? Because if both are True, you raise an error if I understand correctly

Reply:
Ah, yes, exactly, it was.